core/src/main/java/tc/oc/pgm/goals/ObjectiveOption.java

```diff
@@ -0,0 +1,19 @@
+package tc.oc.pgm.goals;
+
+public enum ObjectiveOption {
```

Comment:
```suggestion
public enum ShowOption {
```

Reply:
Agreed! I went ahead and renamed `ObjectiveOption` to `ShowOption` 👍